test/drafts/Strings.test.js

```diff
@@ -0,0 +1,49 @@
+const { constants } = require('openzeppelin-test-helpers');
+
+const StringsMock = artifacts.require('StringsMock');
+
+contract('Strings', function () {
+  const a = 'AAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
+  const b = 'BBBBBBBBBBBBBBBBBBBBBBB';
+  const ab = a + b;
+  const aa = a + a;
+  const empty = '';
+  const zero = 0;
+  const positiveNumber = 3195;
+
+  beforeEach(async function () {
+    this.strings = await StringsMock.new();
+  });
+
+  describe('concatenate', function () {
```

Comment:
I forgot about this before, but we should probably also add tests for UTF-8 strings.

Reply:
I'm not sure what would make good test strings.
I have tried emoji